hedera-node/docs/design/services/smart-contract-service/accounts-blocklisting.md

```diff
@@ -0,0 +1,68 @@
+# Accounts Blocklisting
+
+## Purpose
+
+In Ethereum, everyone that knows a private key is able to authorize a transaction from the corresponding EVM public address, and the first to send a transaction would be eligible to withdraw any funds from that address.
+In Hedera, it is possible for an account with associated EVM address alias to have a key that does not correspond to the alias. This is what we refer to as “EVM address squatting” i.e. using a key that’s different from the key that’s used to derive the EVM address alias from.
+
+It is possible to “squat” on an EVM address in a couple of ways:
+1. Create an account with an alias and then update the account key.
+2. Create an account with an alias and a key that’s different from the key that’s used to derive the alias from.
+
+In both cases, this results in the private key losing control over the EVM address derived from the corresponding public key.
```

Comment:
which "private key" has lost "control"?  I think this means the EVM address' private key is no longer in _sole_ control - that there are now _two_ keys that can drain the address (this is the "first to send a transaction" in the previous sentence but with _two_ keys available instead of just one (shared) one.

Reply:
Not really, the "EVM address' private key" can no longer be used to control the account if the account has a different key - only the key set on the account can be used to sign transactions that transfer funds out of that account. 
Only if the account is updated with a 1 of 2 threshold key with (1) the new account key and (2) "EVM address' private key", then both keys could be used to control the account.